services/cinema_hall.py

```diff
@@ -0,0 +1,19 @@
+from django.db.models import QuerySet
+
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> QuerySet:
+    return CinemaHall.objects.all()
+
+
+def create_cinema_hall(
+        hall_name: str,
+        hall_rows: int,
+        hall_seats_in_row: int
+) -> QuerySet | CinemaHall:

```

Comment:
The return type hint should be `CinemaHall` instead of `QuerySet | CinemaHall` because `CinemaHall.objects.create()` returns a single `CinemaHall` instance, not a QuerySet.